k8s.gcr.io/images/k8s-staging-build-image/images.yaml

```diff
@@ -458,6 +459,15 @@
     "sha256:ef3d86412d9335e7d587aebef9bc004bd42845516fee11e830a49d5e55aaaad0": ["v1.15.10-legacy-1"]
     "sha256:f393f7b488a9488fc1dff42ba24e2674f9a2d962c7dedd93a457aaf414ab956e": ["v1.15.15-legacy-1"]
     "sha256:f9f569e6e1da0fa6ac441b625e599bead8ab10f480ad2a84966cd4a0091e76a8": ["v1.12.17-2"]
+- name: kube-cross-amd64
```

Comment:
do we need those?

Reply:
Hehe yep, I'm stripping them out locally right now :)